CONTRIBUTING.md

```diff
@@ -38,16 +38,17 @@ it is important to understand the structure of the code and the repository.
     These are organized in subdirectories according to what they test:
     see the "Setup" section below for more details.
 
--   Documentation is found in `doc/`, and its structure is described in [its README file](doc/README.md).
+-   Documentation is found in `doc/`, and its structure is described in its README file.
```

Comment:
Could we update the readme to have the new doc build process. 
And add a github action that runs all the commands required to build the docs and save them as an artifact.